packages/labs/analyzer/src/test/server/utils.ts

```diff
@@ -250,9 +228,17 @@ export const setupAnalyzerForNodeTest = (lang: Language, pkg: string) => {
   };
 };
 
-export interface AnalyzerModuleTestContext extends AnalyzerTestContext {
-  module: Module;
-}
+export const setupAnalyzerForNodeTestWithModule = (
```

Comment:
Now that the migration is complete, consider renaming to just `setupAnalyzerForTestWithModule`

Reply:
Oops! I meant to do that :)